packages/core/src/agent/agent.ts

```diff
@@ -1693,17 +1696,22 @@ export class Agent<TAgentId extends string = string, TTools extends ToolsInput =
               });
 
               let result: any;
+              const slugify = require('@sindresorhus/slugify');
```

Comment:
Hmm, doesn't feel like a best practice to import this here, nor to use require for this.